packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -181,6 +181,28 @@ QUnit.module('Chat', moduleConfig, () => {
 
             assert.deepEqual(messageList.option('items'), newItems, 'items value is updated');
         });
+
+        QUnit.test('Chat should pass showDayHeaders to messageList on init', function(assert) {
+            this.reinit({
+                showDayHeaders: false
+            });
+
+            const messageList = MessageList.getInstance(this.$element.find(`.${CHAT_MESSAGELIST_CLASS}`));
```

Comment:
Let's add a helper function to get the messageList instance in the module's before section. It increases readability and reduces duplication because we currently do this in each test within the module.

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925